HackIllinois/Misc/Constants.swift

```diff
@@ -11,6 +11,8 @@
 //
 
 import Foundation
+import CoreData
+import HIAPI
```

Comment:
Looks like these are extra/unused?

Reply:
HIAPI is used for the call to get the times from api